django/db/migrations/state.py

```diff
@@ -6,7 +6,12 @@
 from django.apps import AppConfig
 from django.apps.registry import Apps, apps as global_apps
 from django.conf import settings
+from django.core.exceptions import FieldDoesNotExist
 from django.db import models
+from django.db.migrations.operations.utils import (
+    field_is_referenced, get_references,
+)
```

Comment:
Might be worth moving theses utils to `django.db.migrations.utils` instead since they are used outside of `.operations` now.

Reply:
`field_is_referenced` internally uses and `get_references` internally use `field_references`. The `field_references` is used in operations.fields.py. So would it be beneficial to move the 3 functions to `django.db.migrations.utils` and changing imports for `field_references` in all operations.

Reply:
Since most of the logic related to field referencing will be used by `django.db.migration.state` I think it would make sense to fold/merge `django.db.migrations.operations.utils` into `django.db.migrations.utils`.

Reply:
I have commit the desired changes